agent/agent.go

```diff
@@ -534,7 +534,13 @@ func (a *Agent) reloadWatches(cfg *config.RuntimeConfig) error {
 	// Compile the watches
```

Comment:
@HadarGreinsmark This is ready to merge, can you also add a section describing the new http handler to https://github.com/hashicorp/consul/blob/master/website/source/docs/agent/watches.html.md and I can take it from there after that. 
Thanks for working on this.